### PR TITLE
Event handling

### PR DESCRIPTION
**Pull request by tingamapuro04**
I have worked on the eventHandling branch where i have used big.js as a depedency together with the logic part from calculate.js and operate.js to to enable my calculator perform operations as defined in the calculate.js by adding event handlers to the different components.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,23 +1,72 @@
 /*eslint-disable*/
+import { calculateNewValue } from '@testing-library/user-event/dist/utils';
 import React from 'react';
+import calculate from '../logic/calcualte'
```
Comment (ShahierNashaat) [inline src/components/calculator.js]: - [ ] Kindly fix the spelling in the importing of the calculate file as l in logic has to capital it should be like this `import calculate from '../Logic/calcualte'`.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @tingamapuro04 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
I have changed the folder name from logic to Logic

File: src/components/calculator.js (hunk 1 above)
Comment (danmainah) [reply src/components/calculator.js]: @tingamapuro04 great work implementing the changes. kindly check your spelling its suppose to be a capital `L` for `Logic` not `logic`.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @tingamapuro04, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
I have changed the folder name from logic to Logic

**Review by Ismailco — COMMENTED** [review]
### Hi @tingamapuro04, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**